modules/deploy/pages/deployment-option/self-hosted/kubernetes/redpanda-cluster-recommendations.adoc

```diff
@@ -0,0 +1,91 @@
+= Redpanda Cluster Recommendations
+:description: A list of recommendations for deploying the Redpanda Helm chart in production.
+:tags: ["Kubernetes"]
+:page-aliases: deploy:deployment-option/self-hosted/kubernetes/kubernetes-best-practices.adoc
+
+This topic provides recommendations for deploying Redpanda in production.
```

Comment:
Suggest adding something about Kubernetes in this intro sentence, or duplicating description sentence here. Users would see in the nav tree that this file is under Kubernetes, but otherwise there's nothing about it in the title or intro.